src/app/space/app-launcher/app-launcher.module.ts

```diff
@@ -2,10 +2,20 @@ import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { Http } from '@angular/http';
 
+import {
+  AuthHelperService,
+  Config,
+  HelperService,
+  TokenProvider
+} from 'ngx-forge';
+import { AUTH_API_URL, AuthenticationService } from 'ngx-login-client';
```

Comment:
`AUTH_API_URL` isn't used

Reply:
Have handled it here https://github.com/fabric8-ui/fabric8-ui/pull/2615 @joshuawilson